mp/src/public/mathlib/vector.h

```diff
@@ -374,8 +374,9 @@ class TableVector
 class ALIGN16 VectorAligned : public Vector
 {
 public:
-	inline VectorAligned(void) {};
+	inline VectorAligned(void):w(0) {};
```

Comment:
Leaving `w` uninitialized is problematic when the `VectorAligned` is copied, since uninitialized data would be read, which means undefined behavior occurs.
Perhaps `w` could just be removed, it seems totally unused. Maybe it's required for `ALIGN16` to work right? It seemed safest to just initialize it with some dummy value.